Dockerfile

```
@@ -16,6 +16,10 @@ RUN bun run build
 FROM base AS final
 ARG BUILD_SHA
 ENV BUILD_SHA=${BUILD_SHA} NODE_ENV=production PORT=8080
+# tini reaps orphans. PID 1 is the cluster supervisor (a normal process, so the
+# kernel does not auto-reap for it) and Bun has no waitpid binding, so every
+# HEALTHCHECK `wget` would otherwise leak a zombie forever.
+RUN apk add --no-cache tini
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)